covalent/_workflow/lepton.py

```diff
@@ -285,6 +292,11 @@ def shell_wrapper(*args, **kwargs) -> Any:
             import builtins
             import subprocess
 
+            # remove reserved retval kwargs from kwargs in lepton (ex. 'files')
```

Comment:
If `files` is considered an illegal parameter name, wouldn't it be better to fail loudly here?

Reply:
This was a temporary work around which is now removed @cjao , it seems that `DepsCall` return values are not supported in general by Leptons. Since this is out of scope for this PR there will be a separate issue to address this.

For now I am throwing an exception when Leptons are created with `DepsCall` objects that have a truthy `retval_keyword` attribute.